src/Functions/HasTokenImpl.h

```diff
@@ -31,7 +31,8 @@ struct HasTokenImpl
         const ColumnString::Offsets & haystack_offsets,
         const std::string & pattern,
         const ColumnPtr & start_pos,
-        PaddedPODArray<UInt8> & res)
+        PaddedPODArray<UInt8> & res,
+        ColumnUInt8* null_map = nullptr)
```

Comment:
```suggestion
        ColumnUInt8 * null_map = nullptr)
```
(should be like this by our code style)

Reply:
Thanks for noticing! I'll make sure to run clang-format.